### PR TITLE
feat: expose getMediaHandler

### PR DESCRIPTION
**Pull request by ThaUnknown**
i'm like 60% this is sure as it doesn't handle null, I really dont understand napi

### CODE REVIEW

File: src/media-track-wrapper.cpp [hunk 1]
```diff
@@ -256,6 +257,20 @@ void TrackWrapper::setMediaHandler(const Napi::CallbackInfo &info)
...
+Napi::Value TrackWrapper::getMediaHandler(const Napi::CallbackInfo &info)
+{
+    if (!mTrackPtr)
+    {
+        Napi::Error::New(info.Env(), "getMediaHandler() called on destroyed track").ThrowAsJavaScriptException();
+        return info.Env().Null();
+    }
+
+    Napi::Env env = info.Env();
+    std::shared_ptr<rtc::Track> newTrack = mTrackPtr->getMediaHandler();
```
Comment (paullouisageneau) [inline src/media-track-wrapper.cpp:269]: This does not make sense, `getMediaHandler()` does not return a track.

**Review by paullouisageneau — COMMENTED** [review]
`MediaHandler` is just an interface for modules you can plug in tracks, so I'm not sure what you want to achieve by exposing it to JS. Could you please elaborate?

**Comment by ThaUnknown** [comment]
honestly, I don't remember anymore, I wrote this when running on 2 days of no sleep, I don't even remember why I needed this, I think it was because I wanted the underlying `video/audio` to be exposed when ldc called `onTrack`

**Comment by murat-dogan** [comment]
As I understood no need for these changes for now.
If you need it again please create a new PR.
Thanks.

**Pull request closed without merging**